### ISSUE TITLE
how to train the quantized model on custom dataset

### ISSUE THREAD

**Issue by tonyzhangnxp** [mit-han-lab/tiny-training#6]
Hello

I 'm an engineer from NXP. I'm very interesting on your incredible work.
we want to reproduce your works on our platforms.
but I can't find where to training the quantized model.
I checked the mucnet repo, it only contains the model zoo
I also checked the once-for-all repo, but it seems not support tiny subnet yet.

**Comment by tonyzhangnxp** [comment]
@songhan @zhijian-liu @Lyken17 @tonylins 
Sorry to loop you all, is there anybody could answer my question?

Thanks.
BR
Tony

**Comment by loveindex** [comment]
> Hello
> 
> I 'm an engineer from NXP. I'm very interested in your incredible work. I'd read your paper "On-Device Training Under 256KB Memory", we want to reproduce your works on our platforms. but I can't find where to training the quantized model on custom dataset. I checked the mucnet repo, it only contains the model zoo, I can't find the training scripts I also checked the once-for-all repo, but it seems not support tiny subnet yet.
> 
> BR Tony

I also can't find the training code. In the train project, I find the train function in "main.c", it calls "invoke(label)" but "label" is not used in "invoke" function.
